docs/root/version_history/current.rst

```diff
@@ -49,6 +49,7 @@ New Features
 * config: added :ref:`version_text <config_cluster_manager_cds>` stat that reflects xDS version.
 * decompressor: generic :ref:`decompressor <config_http_filters_decompressor>` filter exposed to users.
 * dynamic forward proxy: added :ref:`SNI based dynamic forward proxy <config_network_filters_sni_dynamic_forward_proxy>` support.
+* dynamic forward proxy: added configurable :ref:`circuit breakers <dns_cache_circuit_breakers>` for resolver on DNS cache. This behavior is disabled by the runtime feature `envoy.reloadable_features.disallow_dns_cache_circuit_breakers`.
```

Comment:
```suggestion
* dynamic forward proxy: added configurable :ref:`circuit breakers <dns_cache_circuit_breakers>` for resolver on DNS cache. This behavior can be temporarily disabled by the runtime feature `envoy.reloadable_features.disallow_dns_cache_circuit_breakers`.
```

Also say what happens if disable.

Also, please add a deprecated note below about the cluster circuit breakers.